src/FunderMaps.Core/Entities/InquirySample.cs

```diff
@@ -1,4 +1,4 @@
-﻿using FunderMaps.Core.Types;
+using FunderMaps.Core.Types;
```

Comment:
What happened here?

Reply:
UTF w/o BOM

Reply:
Files on origin were encoded in UTF-8 with BOM. This changes the file to UTF-8 without BOM.